index.html

```diff
@@ -6,6 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Profile</title>
```

Comment:
As highlighted by the previous reviewer, a more descriptive title is better for web accessibility. For example, a screenshot of an open tab with your portfolio looks like this.

<img width="257" alt="portfolio" src="https://user-images.githubusercontent.com/71434803/127240604-9b61fc79-34bf-44a3-8f42-194e5e301785.png">

It would be better to describe the content of the page for example by changing it to `Obote's portfolio`. Kindly implement this.

Reply:
I have changed the title of the page to more meaningful heading.